config.json

```diff
@@ -61,6 +61,8 @@
     "show_accept_visa": false,
     "product_list_display_mode": "grid",
     "logo-position": "center",
+    "logoSize": "250x100",
+    "productImageSize": "300x300",
```

Comment:
Does this work by default? I notice you are using these in the getImage helper, but the values here do not exist in `_images` as an object with `width` and `height`